http/api.http

```diff
@@ -0,0 +1,561 @@
+
+  {
+    "settings": {
+        "debug": false,
+        "disable_review": false,
+        "disable_release_notes": false,
+        "max_files": 150,
+        "review_simple_changes": true,
+        "review_comment_lgtm": false,
+        "path_filters": [
+            "!dist/**",
+            "!**/*.pb.go",
+            "!**/*.lock",
+            "!**/*.yaml",
+            "!**/*.yml",
+            "!**/*.cfg",
+            "!**/*.toml",
+            "!**/*.ini",
+            "!**/*.mod",
+            "!**/*.sum",
+            "!**/*.work",
+            "!**/*.json",
+            "!**/*.mmd",
+            "!**/*.svg",
+            "!**/*.png",
+            "!**/*.dot",
+            "!**/*.md5sum",
+            "!**/*.wasm",
+            "!**/gen/**",
+            "!**/_gen/**",
+            "!**/generated/**",
+            "!**/vendor/**"
+        ],
+        "system_message": "You are `@openai` (aka `github-actions[bot]`), a language model trained by OpenAI. Your purpose is to act as a highly experienced software engineer and provide a thorough review of the code hunks and suggest code snippets to improve key areas such as:\n- Logic\n- Security\n- Performance\n- Data races\n- Consistency\n- Error handling\n- Maintainability\n- Modularity\n- Complexity\n- Optimization\n\nRefrain from commenting on minor code style issues, missing comments/documentation, or giving compliments, unless explicitly requested. Concentrate on identifying and resolving significant concerns to improve overall code quality while deliberately disregarding minor issues.\n\nNote: As your knowledge may be outdated, trust the user code when newer APIs and methods are seemingly being used.",
+        "openai_light_model": "gpt-3.5-turbo",
+        "openai_heavy_model": "gpt-3.5-turbo",
+        "openai_model_temperature": 0,
+        "openai_retries": 5,
+        "openai_timeout_ms": 360000,
+        "openai_concurrency_limit": 4,
+        "summary_token_limits": "max_tokens=4000, request_tokens=2900, response_tokens=1000",
+        "review_token_limits": "max_tokens=4000, request_tokens=2900, response_tokens=1000",
+        "api_base_url": "https://api.openai.com/v1",
+        "github_event_name": "pull_request"
+    },
+    "context": {
+        "eventName": "pull_request",
+        "payload": {
+  "action": "opened",
+  "number": 1,
+  "pull_request": {
+    "url": "https://api.github.com/repos/Infracloud-harsh/coderabbit-test/pulls/1",
+    "id": 1583785059,
+    "node_id": "PR_kwDOKoOXJs5eZqRj",
+    "html_url": "https://github.com/Infracloud-harsh/coderabbit-test/pull/1",
+    "diff_url": "https://github.com/Infracloud-harsh/coderabbit-test/pull/1.diff",
+    "patch_url": "https://github.com/Infracloud-harsh/coderabbit-test/pull/1.patch",
+    "issue_url": "https://api.github.com/repos/Infracloud-harsh/coderabbit-test/issues/1",
+    "number": 1,
+    "state": "open",
+    "locked": false,
+    "title": "30 days of js",
+    "user": {
+      "login": "Infracloud-harsh",
```

Comment:
If committed, this is pretty much hardcoded. So, any other collaborator will be confused about how to interpret / use this.